packages/desktop/electron/lib/Ledger.js

```diff
@@ -3,7 +3,7 @@ const TransportSpeculos = require('@ledgerhq/hw-transport-node-speculos').defaul
 const Iota = require('hw-app-iota').default
 const { ipcRenderer, remote } = require('electron')
 
-const USE_SIMULATOR = false
+const USE_SIMULATOR = true
```

Comment:
@laumair do we want to set this to true?

Reply:
Before next release, yes.